### PR TITLE
Fixed trivia-bot

### PR DESCRIPTION
**Pull request by Vambat-01**

### CODE REVIEW

File: trivia-bot/Dockerfile [hunk 1]
```diff
@@ -11,4 +11,4 @@ COPY ./main.py /app/main.py
 
 WORKDIR /app
 
-CMD ["python", "./main.py", "-config", "resources/config_server.json"]
+CMD ["python", "./main.py", "-config", "resources/config_server_aws.json"]
```
Comment (andrewvorobyev) [inline trivia-bot/Dockerfile:14]: You should not commit that. I think we can run container with a custom command on `AWS`. See https://docs.docker.com/compose/compose-file/compose-file-v3/#command to modify `docker-compose-aws`.

File: trivia-bot/trivia/bot_state.py [hunk 2]
```diff
@@ -71,11 +71,12 @@ def create_in_game_state(self):
         for question in game_questions:
             indexed_answers = list(enumerate(question.answers))
             self.random.shuffle(indexed_answers)
-            correct_answer = question.correct_answer
+            cor_answer = question.correct_answer
```
Comment (andrewvorobyev) [inline trivia-bot/trivia/bot_state.py:74]: bad variable name. Rename to `original_correct_answer`

**Review by a-and-v — CHANGES_REQUESTED** [review]
.

File: trivia-bot/trivia/bot_state.py [hunk 3]
```diff
@@ -71,11 +71,12 @@ def create_in_game_state(self):
...
+            correct_answer = 0
             answers = []
 
             for (index, (original_index, answer)) in enumerate(indexed_answers):
-                if original_index == 0:
+                if original_index == cor_answer:
```
Comment (andrewvorobyev) [inline trivia-bot/trivia/bot_state.py:79]: Why none of your tests caught this error? Add a test that will verify this logic.

File: trivia-bot/test/test_game_state_factory.py [hunk 4]
```diff
@@ -21,6 +21,23 @@ def test_answer_shuffling_for_in_game_state(self):
                           ]
         self.assertEqual(questions_list, actual.state.questions)
 
+    def test_answer_shuffling_in_game_state(self):
```
Comment (andrewvorobyev) [inline trivia-bot/test/test_game_state_factory.py:24]: how is this test different from the test above?

File: trivia-bot/trivia/question_storage.py [hunk 5]
```diff
@@ -223,6 +223,25 @@ def add_questions(self, questions: List[Question]):
         self.connection.commit()
 
 
+class InMemoryQuestionStorage(QuestionStorage):
+    """
+        Класс для теста, считывает вопросы и списка вопросов
+    """
+    def __init__(self, list_questions: List[Question]):
+        """
+            Передает список вопросов
```
Comment (andrewvorobyev) [inline trivia-bot/trivia/question_storage.py:232]: куда передает? кто и кому?

File: trivia-bot/trivia/question_storage.py [hunk 6]
```diff
@@ -223,6 +223,25 @@ def add_questions(self, questions: List[Question]):
...
+        :param list_questions: спислк вопросов
+        """
+        self.list_questions = list_questions
+
+    def load_questions(self) -> List[Question]:
+        """
+            Возаращает переданные попросы из списка
```
Comment (andrewvorobyev) [inline trivia-bot/trivia/question_storage.py:239]: You can remove that since it's an interface method. The doc is present on the interface.

**Review by a-and-v — CHANGES_REQUESTED** [review]
Looks good! Please see the comments.

File: trivia-bot/trivia/question_storage.py [hunk 7]
```diff
@@ -223,6 +223,24 @@ def add_questions(self, questions: List[Question]):
         self.connection.commit()
 
 
+class InMemoryQuestionStorage(QuestionStorage):
+    """
+        Класс для теста, считывает вопросы и списка вопросов
```
Comment (andrewvorobyev) [inline trivia-bot/trivia/question_storage.py:228]: It's not for test only. It's a proper storage that can be used anywhere. The docs for it should not use `test` word.

**Review by a-and-v — APPROVED** [review]

**Pull request merged**